man/networkd.conf.xml

```diff
@@ -276,9 +287,10 @@ DUIDRawData=00:00:ab:11:f9:2a:c2:77:29:f9:5c:00</programlisting>
       <varlistentry>
         <term><varname>DUIDType=</varname></term>
         <term><varname>DUIDRawData=</varname></term>
+        <term><varname>DefaultUseDomains=</varname></term>
         <listitem><para>As in the [DHCPv4] section.</para>
 
-        <xi:include href="version-info.xml" xpointer="v249"/></listitem>
+        <xi:include href="version-info.xml" xpointer="v256"/></listitem>
```

Comment:
Please do not merge the entry with `DUIDType=` and `DUIDRawData=`, to keep the version info for them.